app/views/user/orders/index.html.erb

```diff
@@ -1,6 +1,11 @@
-<h1>My Orders</h1>
+<h1><%= "#{current_user.name}'s Orders" %></h1>
 
-<% @user_orders.each do |order| %>
-  <h3><%= link_to "Order #{order.id}" %></h3>
-  <p><%= order.status.titleize %></p>
+<% current_user.orders.each do |order| %>
```

Comment:
Ideally, we should not make a query such as current_user.orders in the view. We have an instance variable built out from the previous PR @user_orders.
We should use @user_orders to comply with MVC.

Reply:
Good catch! Fixed on branch.